<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>503</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>送餐系统服务器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/Welcome.png</normaloff>:/image/Welcome.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>本地状态</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>本机信息</string>
     </property>
     <widget class="QPushButton" name="btnMoreMessage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查看更多信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtLocalIP">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>本主机名：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtLocalName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>391</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>网络连接</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>本地网络状态：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtLocalNet">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>已连接机器数：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtLocalClient">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>端口号设置：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPort">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>8010</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>已连接的机器：</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listClient">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>状态区域</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>命令区域</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>更新与下载</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更新软件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoteDown">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>远程下载</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
